source/dub/generators/build.d

```diff
@@ -577,8 +577,16 @@ else version (FreeBSD)
 	 version = KQUEUE;
 else version (OSX)
 	 version = KQUEUE;
-else
-	 static assert(0, "Filewatcher support not yet implemented");
```

Comment:
Please keep that else branch and explicitly add `else version (Windows)` above, there are other platforms still not supported.